http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -239,7 +247,7 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
       .parTraverseN(parallelism) { case (runtime, image, container) =>
         for {
           ctx <- ev.ask
-          isUp <- container.isProxyAvailable(runtime.cloudContext, runtime.runtimeName)
+          isUp <- container.isProxyAvailable(runtime.cloudContext, runtime.runtimeName).handleError(_ => false)
```

Comment:
I _think_ this was the metrics collection issue. We were getting some exceptions like
```
javax.net.ssl.SSLHandshakeException: No trust manager was able to validate this certificate chain: # of exceptions = 2
```
from runtimes, which weren't being caught and caused the whole metrics collection process to crash.

…I'm still trying to validate this though.

Reply:
Good catch, I would not have expected this to crash the entire metrics collection though, just the runtimes 🤔

Reply:
That's the weird part, I would also expect to see a log line ["Unexpected error occurred during metric monitoring"](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala#L55), but I'm not seeing it. 🤔 
Wondering if the SSL error is causing other issues, this is what I want to dig in a little more on a BEE, and hopefully replicate.